docs/en/tutorials/config.md

```diff
@@ -395,10 +395,16 @@ lr_config = dict(  # Learning rate scheduler config used to register LrUpdater h
     step=[24, 32])  # Steps to decay the learning rate
 checkpoint_config = dict(  # Config of set the checkpoint hook, Refer to https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/checkpoint.py for implementation.
     interval=1)  # The save interval is 1
-log_config = dict(  # config of register logger hook
+log_config = dict(  # Config to register logger hook
     interval=50,  # Interval to print the log
-    hooks=[dict(type='TextLoggerHook'),
-           dict(type='TensorboardLoggerHook')])  # The logger used to record the training process.
+    hooks=[
+      dict(type='TextLoggerHook', by_epoch=False),
+      dict(type='TensorboardLoggerHook', by_epoch=False),
+      dict(type='WandbLoggerHook', by_epoch=False,  # The Wandb logger is also supported, It requires `wandb` to be installed.
+           init_kwargs={'entity': "OpenMMLab",  # The entity used to log on Wandb
+                        'project': "MMTracking",  # Project name in WandB 
```

Comment:
shall we rename mmtracking to MMDet3D?

Reply:
Sure, thanks for mentioning, I made a few changes and fix that.